llvm_jni/modules/jni.llvm/Builder.bal

```diff
@@ -76,7 +76,39 @@ public distinct class Builder {
         }
     }
 
-    public function binaryInt(BinaryIntOp op, Value lhs, Value rhs, string? name = ()) returns Value {
+    public function iArithmeticNoWrap(IntArithmeticOp op, Value lhs, Value rhs, string? name=()) returns Value {
+        string regName = self.extractName(name);
+        match op {
+            "add" => {
+                Value val = new (jLLVMBuildNSWAdd(self.LLVMBuilder, lhs.LLVMValueRef, rhs.LLVMValueRef, java:fromString(regName)));
```

Comment:
Does LLVMBuildNSW* add both `nsw` and `nuw`?

Reply:
No unfortunately with the C-API as far as I can work out we can do only `nsw` or `nuw` not both